### PR TITLE
Integrate Erik and fix PublicationModel.set issue

### PR DESCRIPTION
**Pull request by chuym**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+/* jshint node:true */
```
Comment (chuym) [inline gulpfile.js]: @spencer-brown - I based off from `users` module for this file. I removed some configs like `handlebars` and `common` that is not needed here, could you take a quick look?

**Comment by chuym** [comment]
I'll merge to test out travis integration but won't publish until reviewed

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+const build = new MultiBuild({
+  gulp,
+  targets: _.compact([
+    'index',
+    SHOULD_BUILD_TESTS && 'tests'
+  ]),
+  entry: (target) => (target === 'tests') ? TEST_FILES : `src/${target}.js`,
+  rollupOptions: (target) => {
+    return {
+      external: ['jquery', 'backbone', 'underscore'],
```
Comment (ttacon) [inline gulpfile.js]: @chuym, you'll want to pull in the external rollup config [here](https://github.com/mixmaxhq/backbone-publication/blob/master/rollup.config.js) (or internalize it for the build of this module using a new `gulp` task on for the `npm` `build` script target).

**Comment by chuym** [comment]
@wearhere I get what happened now Jeff, so this was actually working as intended, but out `change` we trigger in here: https://github.com/mixmaxhq/backbone-publication/blob/706b7f1d52702e59c8f9f1120b6366e24d863691/src/PublicationModel.js#L89 is different that the `change` event triggered by Backbine here: https://github.com/jashkenas/backbone/blob/master/backbone.js#L532

We are sending the attribute changes, whereas backbone does not.

This is change is very subtle if you are not familiar with the regular `change` event signature (Sadly, I wasn't aware of it) and I relied on `change` events containing the new attributes, but I didn't realize that it was _our_ own custom event that was doing this. I've been relying on that for other code, I think we should be more careful when overriding libraries behaviors so we keep the same interface or implement our events instead overloading existing ones.

I will revert the PublicationModel change but keep erik + travis setup.

**Pull request merged**